src/Components/v2/Carousel.tsx

```diff
@@ -265,18 +279,25 @@ export class BaseCarousel extends React.Component<
   }
 
   renderRightArrow = () => {
-    const { onArrowClick, renderRightArrow, showArrows } = this.props
+    const { onArrowClick, renderRightArrow, showArrows, height } = this.props
 
     if (!showArrows) {
       return null
     }
 
+    const rightPosition =
+      this.props.arrowPosition !== null &&
+      this.props.arrowPosition !== undefined
```

Comment:
A shortcut to check null or undefined in one line is `this.props.arrowPosition != null` (single equals)

Reply:
ah and this checks `undefined` too? interesting

Reply:
lil js hack 😄